8bitzz/COSC2101
# Integrate Docker for FrontEnd and Backend

### Changelog
- Support Docker for Frontend and backend service
- Use docket-compose to start the service

### How to test
1. Download and Install Docker at https://www.docker.com/get-started
2. Open Docker app and make sure it works
3. Check out this branch
4. At Root directory, run `docker-compose up -d`
5. Wait until it's done 
6. Open http://localhost:300 with Google Chrome and verify it works.
